### PR TITLE
add favorite prescriptions to patient grouped by user

### PR DESCRIPTION
**Pull request by scorpions786**
**maybe need some optimization and UI Improvement **

### CODE REVIEW

File: templates/prescription/general_list.html [hunk 1]
```diff
@@ -131,7 +131,7 @@
 
 <div id="prescription_list">
 
-<form name="presc">
+<form name="presc" method="post" action="{$WEBROOT}/add_rxs_to_list.php">
```
Comment (bradymiller) [inline templates/prescription/general_list.html]: hi @scorpions786 ,
Thanks for the contribution.
Is the above script add_rxs_to_list.php missing?
-brady

File: sql/5_0_0-to-5_0_1_upgrade.sql [hunk 2]
```diff
@@ -559,6 +559,16 @@ ALTER TABLE `pharmacies` ADD `npi` INT(12) DEFAULT NULL;
...
 #EndIf
 
+#IfNotTable prescription_fav_list
+CREATE TABLE `prescription_fav_list` (
+  `presc_id` int(11) DEFAULT NULL,
+  `provider_id` int(11) DEFAULT NULL,
+  `list_id` int(11) NOT NULL AUTO_INCREMENT,
+  `list_name` varchar(255) NOT NULL,
+    PRIMARY KEY (`list_id`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
```
Comment (bradymiller) [inline sql/5_0_0-to-5_0_1_upgrade.sql]: Also place this new table in the sql/database.sql script

File: templates/prescription/general_list.html (hunk 1 above)
Comment (scorpions786) [reply templates/prescription/general_list.html]: yeah, I missed that file -
just added

**Comment by scorpions786** [comment]
suggest me the location of these files

File: sql/database.sql [hunk 3]
```diff
@@ -6251,6 +6251,19 @@ CREATE TABLE `prescriptions` (
 -----------------------------------------------------------
 
 --
+-- Table structure for table `prescriptions`
+--
+DROP TABLE IF EXISTS `prescriptions`;
```
Comment (bradymiller) [inline sql/database.sql]: wrong name above
``````
DROP TABLE IF EXISTS `prescription_fav_list`;
``````

**Comment by bradymiller** [comment]
hi @scorpions786 ,
Regarding 'suggest me the location of these files' comment, which files are you looking for?

**Comment by scorpions786** [comment]
"add_rxs_to_list.php"

**Pull request closed without merging**